ush/set_fv3nml_ens_stoch_seeds.py

```diff
@@ -1,32 +1,28 @@
 #!/usr/bin/env python3
```

Comment:
This file was linted to PEP8 standards adopted by SRW. That includes its new lower case naming convention required of Python modules.